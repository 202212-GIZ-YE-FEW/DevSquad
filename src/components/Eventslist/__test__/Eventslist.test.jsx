import renderer from "react-test-renderer";

import Eventslist from "../index";

it("renders correctly", () => {
    const tree = renderer.create(<Eventslist />).toJSON();
    expect(tree).toMatchSnapshot();
});

jest.mock("next/router", () => ({
    useRouter() {
        return {
            route: "/",
            pathname: "",
            query: "",
            asPath: "",
        };
    },
}));
