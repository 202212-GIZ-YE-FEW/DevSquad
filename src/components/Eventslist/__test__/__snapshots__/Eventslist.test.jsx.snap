// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="md:m-8 m-2 font-Rubik"
>
  <div
    className="font-Rubik flex flex-col items-center py-10 "
  >
    <div
      className="w-full fixed top-0 h-[50%] hidden"
      onClick={[Function]}
    />
    <p
      className="text-5xl md:font-extrabold font:medium "
    >
      Welcome, 
      asd
      !
    </p>
    <p
      className="font-normal text-start"
    >
      Explore and Join Events.
    </p>
  </div>
  <div
    className="flex justify-evenly md:pb-0 pb-4 pt-4 border-t-2 border-b-2 border-black md:border-0 sm:hidden"
  >
    <button
      className="flex items-center p-1 border border-black rounded mr-1"
      onClick={[Function]}
    >
      <p
        className="pr-1 font-Rubik text-xs"
      >
        Change Interest
      </p>
      <svg
        fill="none"
        height="8"
        viewBox="0 0 13 8"
        width="13"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M11.4704 0.705392C11.6865 0.465228 12.0564 0.445759 12.2965 0.661906C12.5366 0.878053 12.5561 1.24797 12.34 1.48813L7.07538 7.3385C6.84301 7.59672 6.43816 7.59672 6.20579 7.3385L0.941182 1.48813C0.725065 1.24797 0.744531 0.878053 0.984662 0.661906C1.22479 0.445759 1.59465 0.465228 1.81077 0.705392L6.64058 6.07259L11.4704 0.705392Z"
          fill="#1A1A1A"
        />
      </svg>
    </button>
    <button
      className="flex items-center  p-1 border border-black rounded md:hidden mr-1"
      onClick={[Function]}
    >
      <p
        className="pr-1 font-Rubik text-xs"
      >
        Change Location
      </p>
      <svg
        fill="none"
        height="8"
        viewBox="0 0 13 8"
        width="13"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M11.4704 0.705392C11.6865 0.465228 12.0564 0.445759 12.2965 0.661906C12.5366 0.878053 12.5561 1.24797 12.34 1.48813L7.07538 7.3385C6.84301 7.59672 6.43816 7.59672 6.20579 7.3385L0.941182 1.48813C0.725065 1.24797 0.744531 0.878053 0.984662 0.661906C1.22479 0.445759 1.59465 0.465228 1.81077 0.705392L6.64058 6.07259L11.4704 0.705392Z"
          fill="#1A1A1A"
        />
      </svg>
    </button>
    <button
      className="flex items-center p-1 border border-black rounded sm:hidden"
      onClick={[Function]}
    >
      <p
        className="pr-1 font-Rubik text-xs"
      >
        Change Date
      </p>
      <svg
        fill="none"
        height="8"
        viewBox="0 0 13 8"
        width="13"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M11.4704 0.705392C11.6865 0.465228 12.0564 0.445759 12.2965 0.661906C12.5366 0.878053 12.5561 1.24797 12.34 1.48813L7.07538 7.3385C6.84301 7.59672 6.43816 7.59672 6.20579 7.3385L0.941182 1.48813C0.725065 1.24797 0.744531 0.878053 0.984662 0.661906C1.22479 0.445759 1.59465 0.465228 1.81077 0.705392L6.64058 6.07259L11.4704 0.705392Z"
          fill="#1A1A1A"
        />
      </svg>
    </button>
  </div>
  <div
    className="sm:grid sm:grid-cols-3 md:gap-6 sm:gap-3"
  >
    <div
      className="flex flex-col sm:items-center sm:space-y-6"
    >
      <div
        className="sm:block sm:static sm:z-0
                        hidden
                        "
      >
        <div
          className="fixed sm:static sm:z-0 z-50 top-1/2 left-1/2 sm:transform-none transform -translate-x-1/2 bg-white w-full sm:w-0 sm:h-full h-[80vh] shadow-inner sm:shadow-none shadow-gray-700 rounded-lg"
        >
          <div
            className="flex flex-col items-center"
          >
            <p
              className="font-medium font-Rubik text-center block sm:hidden text-lg my-2"
            >
              calendar.date
            </p>
            <div
              className="md:w-64 w-56 border-2 rounded-lg border-black sm:m-3 mb-2"
            >
              <div
                className="grid grid-cols-7 justify-items-center m-3"
              >
                <p
                  className="text-xs font-bold"
                >
                  Mon
                </p>
                <p
                  className="text-xs font-bold"
                >
                  Tue
                </p>
                <p
                  className="text-xs font-bold"
                >
                  Wed
                </p>
                <p
                  className="text-xs font-bold"
                >
                  Thu
                </p>
                <p
                  className="text-xs font-bold"
                >
                  Fri
                </p>
                <p
                  className="text-xs font-bold"
                >
                  Sat
                </p>
                <p
                  className="text-xs font-bold"
                >
                  Sun
                </p>
              </div>
              <div
                className="grid grid-cols-7 gap-2 justify-items-center m-3"
              >
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    01
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    02
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    03
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    04
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    05
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    06
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    07
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    08
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    09
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    10
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    11
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    12
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    13
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    14
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    15
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    16
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    17
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    18
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    19
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    20
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    21
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    22
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    23
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    24
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    25
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    26
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    27
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    28
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    29
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    30
                  </span>
                </label>
                <label>
                  <input
                    hidden={true}
                    onChange={[Function]}
                    onClick={[Function]}
                    type="checkbox"
                  />
                  <span
                    className="bg-white rounded font-bold text-lg"
                  >
                    31
                  </span>
                </label>
              </div>
            </div>
            <div
              className="flex flex-row justify-between md:w-64 w-56 sm:m-3 mb-2"
            >
              <button
                className="border-black border border-b-4 border-r-4 rounded-lg text-base sm:text-lg font-medium font-Rubik md:p-2 p-1"
                onClick={[Function]}
              >
                calendar.thismonth
              </button>
              <button
                className="border-black border border-b-4 border-r-4 rounded-lg text-base sm:text-lg font-medium font-Rubik md:p-2 p-1"
                onClick={[Function]}
              >
                calendar.nextmonth
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        className="sm:block md:w-64 w-56 hidden"
      >
        <div
          className="p-6 sm:p-0 bg-white w-full sm:h-full h-[80vh] fixed sm:static sm:z-0 z-50 top-1/2 left-1/2 sm:transform-none transform -translate-x-1/2 shadow-inner sm:shadow-none shadow-gray-700 rounded-lg"
        >
          <div
            className="border-t-2 border-b-2 border-black h-32 font-medium text-center w-full mt-[20%] sm:mt-0"
          >
            <button
              className="underline p-4"
            >
              Change Location
            </button>
            <p
              className="border-black border border-b-4 border-r-4 rounded-lg text-base sm:text-lg font-medium font-Rubik sm:p-2 p-1 w-full"
            >
              İzmir, TR
            </p>
          </div>
        </div>
      </div>
      <div
        className="sm:block hidden"
      >
        <div>
          <div
            className="md:w-64 sm:w-56 w-full bg-white sm:h-full h-[50%] overflow-auto scrollbar fixed sm:static sm:z-0 z-50 top-1/2 left-1/2 sm:transform-none transform -translate-x-1/2 shadow-inner sm:shadow-none shadow-gray-700 rounded-lg"
          >
            <div
              className="grid grid-cols-1 gap-2 "
            >
              <p
                className="font-medium font-Rubik underline text-center sm:block hidden"
              >
                Pick Your Interest
              </p>
              <p
                className="font-medium font-Rubik text-center block sm:hidden text-lg my-2"
              >
                Change Interset
              </p>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={0}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="All"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    All
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={1}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="No Poverty"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    No Poverty
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={2}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="Zero Hunger"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    Zero Hunger
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={3}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="Good Health And Well-Being"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    Good Health And Well-Being
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={4}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="Quality Education"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    Quality Education
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={5}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="Gender Equality"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    Gender Equality
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={6}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="Clean Water And Sanitation"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    Clean Water And Sanitation
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={7}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="Affordable And Clean Energy"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    Affordable And Clean Energy
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={8}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="Decent Work And Economic Growth"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    Decent Work And Economic Growth
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={9}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="Industry Innovation And Infrastructure"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    Industry Innovation And Infrastructure
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={10}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="Reduced Inequalities"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    Reduced Inequalities
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={11}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="Sustainable Cities And Communities"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    Sustainable Cities And Communities
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={12}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="Responsible Consumption And Production"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    Responsible Consumption And Production
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={13}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="Climate Action"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    Climate Action
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={14}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="Life Below Water"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    Life Below Water
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={15}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="Life On Land"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    Life On Land
                  </span>
                </label>
              </div>
              <div>
                <label
                  className="flex sm:block"
                >
                  <input
                    className="sm:hidden sm:m-0 m-6"
                    id={16}
                    name="types"
                    onChange={[Function]}
                    type="checkbox"
                    value="Peace And Justice And Strong Institutions"
                  />
                  <span
                    className="checked flex items-center justify-center text-center  sm:border-2 sm:border-black border-0 md:p-3 rounded h-16  font-Rubik font-medium sm:text-base text-xs ml-2"
                  >
                    Peace And Justice And Strong Institutions
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="sm:col-span-2 col-span-3"
    >
      <div
        className="md:order-2 order-1 text-center md:text-start"
      >
        <p
          className="py-3 font-Rubik  font-bold"
        >
           Pages
        </p>
        <ul
          className="sm:mx-32 inline-flex max-w-xs justify-center items-center border border-black h-8"
        >
          <li
            className="bg-gray-200 opacity-50 cursor-not-allowed m-1 h-8 pointer-events-none"
            onClick={[Function]}
          >
            <svg
              fill="none"
              height="34"
              viewBox="0 0 34 34"
              width="34"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clip-rule="evenodd"
                d="M0.5 17C0.5 26.1127 7.8873 33.5 17 33.5C26.1127 33.5 33.5 26.1127 33.5 17C33.5 7.8873 26.1127 0.5 17 0.5C7.8873 0.5 0.5 7.8873 0.5 17ZM13.3833 8C13.6175 8 13.8422 8.0879 14.0078 8.24435L24.2413 16.4101C24.5862 16.7359 24.5862 17.2641 24.2413 17.5899L14.0078 25.7556C13.6629 26.0815 13.1036 26.0815 12.7587 25.7556C12.5931 25.5992 12.5 25.387 12.5 25.1657V8.83427C12.5 8.37352 12.8955 8 13.3833 8Z"
                fill="#1A1A1A"
                fill-opacity="0.7"
                fill-rule="evenodd"
              />
            </svg>

          </li>
          <li
            className="bg-gray-200 opacity-50 cursor-not-allowed m-1 h-8 pointer-events-none"
            onClick={[Function]}

          >
            <svg
              fill="none"
              height="34"
              viewBox="0 0 34 34"
              width="34"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clip-rule="evenodd"
                d="M0.5 17C0.5 26.1127 7.8873 33.5 17 33.5C26.1127 33.5 33.5 26.1127 33.5 17C33.5 7.8873 26.1127 0.5 17 0.5C7.8873 0.5 0.5 7.8873 0.5 17ZM9.75871 16.4101L19.9922 8.24435C20.1578 8.0879 20.3825 8 20.6167 8C21.1045 8 21.5 8.37352 21.5 8.83427V25.1657C21.5 25.387 21.4069 25.5992 21.2413 25.7556C20.8964 26.0815 20.3371 26.0815 19.9922 25.7556L9.75871 17.5899C9.41376 17.2641 9.41376 16.7359 9.75871 16.4101Z"
                fill="#1A1A1A"
                fill-opacity="0.7"
                fill-rule="evenodd"
              />
            </svg>
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
